transport.go

```diff
@@ -45,7 +45,9 @@ var (
 	errPeerOptions     = errors.New("do not specify peers using --peer and --peer-list")
 )
 
+// works for either tchannel or grpc
```

Comment:
do we need this to work for grpc? we should listen on all interfaces for grpc, don't think this is needed.